example/App.js

```diff
@@ -111,7 +112,10 @@ export default class App extends Component {
     deviceJSON.phoneNumber = DeviceInfo.getPhoneNumberSync(); // needs android.permission.READ_PHONE_STATE
     deviceJSON.ApiLevel = DeviceInfo.getApiLevelSync();
     deviceJSON.carrier = DeviceInfo.getCarrierSync();
-    deviceJSON.totalMemory = DeviceInfo.getTotalMemorySync();
+    deviceJSON.totalMemory =
+      Platform.OS !== 'windows'
+        ? DeviceInfo.getTotalMemorySync()
+        : 'Not supported';
```

Comment:
This one (and the other one below) are worrisome - there should be no reason to have Platform-specific code in App.js, one of the API rules for the module is that if it's not supported on a platform the API itself will return one of the designated default values:

https://github.com/react-native-community/react-native-device-info/blame/master/README.md#L303

I think what needs to happen is these two methods: 

https://github.com/react-native-community/react-native-device-info/blob/f617863017dce0daf623504e7987652561f4ee49/src/index.ts#L918-L948

Need to be changed so that windows falls through to the default case (and returns '-1', the default response when a platform doesn't support a numeric API)